checker/check.go

```diff
@@ -305,16 +306,25 @@ func (triggerChecker *TriggerChecker) check(
 		log := logger.Clone().
 			String(moira.LogFieldNameMetricName, metricName)
 
-		log.Debug().Msg("Checking metrics")
-
 		targets = conversion.Merge(targets, aloneMetrics)
 
 		metricState, needToDeleteMetric, err := triggerChecker.checkTargets(metricName, targets, log)
+		prevMetricState := checkData.Metrics[metricName]
+		isNewMetric := prevMetricState.Timestamp != metricState.Timestamp
+
 		if needToDeleteMetric {
-			log.Info().Msg("Remove metric")
-			checkData.RemoveMetricState(metricName)
-			err = triggerChecker.database.RemovePatternsMetrics(triggerChecker.trigger.Patterns)
+			if metricState.Maintenance == 0 || (metricState.Maintenance != 0 && time.Now().Unix() >= metricState.Maintenance) {
+				log.Info().Msg("Remove metric")
```

Comment:
let's add .String("metric_name", metricName) and make level debug, i think?

Reply:
yep, change it